examples/connectivity/plot_adhd_covariance.py

```diff
@@ -8,11 +8,26 @@
  
 plotted_subject = 0  # subject to plot
 
+import numpy as np
 
 import matplotlib.pyplot as plt
+
+from nilearn import plotting, image
 from nilearn.plotting import cm
 
 
+def plot_connectome(cov, atlas_maps, **kwargs):
+    """Plot connectome given a covariance matrix and atlas maps"""
+    imgs = image.iter_img(atlas_maps)
+    regions_coords = np.array(
+        [plotting.find_xyz_cut_coords(img) for img in imgs])
```

Comment:
> The name of the function is also misleading. We should have a function named
> find_poi or so.

I agree. I am +1 for renaming if there are good suggestion.

> And it could handle 3D and 4D imgs so that a loop is not necessary.

I am -0. I am afraid of errors in user code going unnoticed until too
late to give a good error message.